Pipfile

```diff
@@ -0,0 +1,25 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+
+[dev-packages]
+
+
+
+[packages]
+
+"flake8" = "==3.5.0"
+"flake8-isort" = "==2.3"
+funcsigs = "==1.0.2"
+mozlog = "==3.7"
+PyPOM = "==1.2.0"
+pytest = "==3.3.2"
+pytest-metadata = "==1.5.1"
+pytest-selenium = "==1.11.4"
+pytest-variables = "==1.7.1"
+pytest-xdist = "==1.22.0"
+"querystringsafe_base64" = "==1.0.0"
+selenium = "==3.8.1"
```

Comment:
My basic understanding of the Pipfile is that we should not pin here unless it's necessary. It means we can regenerate the Pipfile.lock at any time from the latest versions of dependencies, but only commit the updated lock file once we're happy that the dependencies are good.